develop/jsapi/create_calcite_maps_app/lab.md

```diff
@@ -1,6 +1,6 @@
 ###Create an ArcGIS JS 4.x - Calcite Maps App with Bootstrap
```

Comment:
not sure how pervasive it is, but i really feel like we should be using `4.0` (not `4.x`)